### PR TITLE
Status update tuning

### PR DESCRIPTION
**Pull request by russgold**
Addresses:
- OWLS-93144: fix conditions for admin-only
- OWLS-95965: domain Completed condition must check Ready (not Running)
- NOJIRA: domain Available condition must check Ready (not Running)

### CODE REVIEW

**Review by tbarnes-us — APPROVED** [review]

**Comment by tbarnes-us** [comment]
**Note:** I see nothing in this change that seems directly related to "Ensure that the 4.0 available condition is false if domain.spec.serverStartPolicy is ADMIN_ONLY" (that's from OWLS-93144).  If this is incorrect, then OWLS-93144 is not completely resolved by this change.

**Comment by russgold** [comment]
> **Note:** I see nothing in this change that seems directly related to "Ensure that the 4.0 available condition is false if domain.spec.serverStartPolicy is ADMIN_ONLY" (that's from OWLS-93144). If this is incorrect, then OWLS-93144 is not completely resolved by this change.

I added unit tests to confirm that behavior; as far as I can tell, it was already working, since the status computation depends on what servers are supposed to be running, which is computed from the start policy.

**Comment by doxiao** [comment]
Can you post Jenkins integration test results?

**Comment by russgold** [comment]
Jenkins run https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/8207

**Review by doxiao — APPROVED** [review]
LGTM

File: operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java [hunk 1]
```diff
@@ -828,19 +828,14 @@ private int minReplicas() {
 
         private long numServersRunning() {
```
Comment (tbarnes-us) [inline operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java:829]: numServersRunning is misnamed - it's now checking ready

**Pull request merged**